### ISSUE TITLE
[map] infinitely sized map or a map that is 5x the size of the humongous map

### ISSUE THREAD

**Issue by Roozz1** [studio-minus/ppg-bugs-and-requests#1391]
**Feature description**
infinitely sized map is just a map that goes on forever, both height and length, no floor, no roof, no walls, just like the void map
if the infinitely sized map cannot be added due to floating point errors or something else, a map that is 5x the size of the humongous map (both height and length) would do the job

**Feature purpose**
a map that is infinitely sized or a map that is 5x the size of the humongous map will be extremely handy for planes, cars, etc
the humongous map is just too small for planes and stuff imo, so a larger map would be quite nice

**Additional context**
not really context, but just a lil note or something:
if you do consider adding an infinitely sized map, maybe replace it with the void map if there wont be a floor

**Comment by YourLocalPlonker** [comment]
The problem with an infinite map is that it would not fit with the lore of the game because every map is supposed to represent a test chamber

**Comment by DindinYT37** [comment]
> The problem with an infinite map is that it would not fit with the lore of the game because every map is supposed to represent a test chamber

The game has no official lore...

**Comment by thatGoober** [comment]
> > The problem with an infinite map is that it would not fit with the lore of the game because every map is supposed to represent a test chamber
> 
> The game has no official lore...

True, but the game references that it is a test lab numerous times. In the map selection screen, the "play" button literally calls the maps chambers.

**Comment by DindinYT37** [comment]
> > > The problem with an infinite map is that it would not fit with the lore of the game because every map is supposed to represent a test chamber
> > 
> > 
> > The game has no official lore...
> 
> True, but the game references that it is a test lab numerous times. In the map selection screen, the "play" button literally calls the maps chambers.

Yeah but I think the main focus of the game is to be good game still, so this might be added anyway. (Though it'd probably lag like crap after some "travelling")

**Comment by howlzy** [comment]
Lol people actually giving this game lore that's hilarious

**Comment by howlzy** [comment]
Just kill people with guns there's ur lore

**Comment by Drogo-man** [comment]
Void?

**Comment by DindinYT37** [comment]
> Void?

That's not an infinite map, it's just a camera limitation instead of solid walls and objects too far out of bounds get deleted.
